packages/react-scripts/config/webpack.config.prod.js

```diff
@@ -210,6 +210,7 @@ module.exports = {
                         importLoaders: 1,
                         modules: true,
                         camelCase: true,
+                        localIdentName: '[name]__[local]___[hash:base64:5]',
```

Comment:
we don't need this in production

Reply:
You sure? Why not? :thinking:

Reply:
Because it makes class names nice for dev, doesn't make sense in prod.